### PR TITLE
MTM-49948: added note, that it is possible to confirm POP by support

### PR DESCRIPTION
**Pull request by vfo-softwareag**

### CODE REVIEW

**Review by sarangovindarajan — APPROVED** [review]

File: content/device-integration/mqtt-bundle/device-certificates.md [hunk 1]
```diff
@@ -403,6 +403,9 @@ The steps for the proof of possession are as follows:
 - The proof of possession is confirmed if the uploaded signed verification code matches the signed verification code expected by the platform:
 ![Proof of possession completed](/images/mqtt/devmgmt-mqtt-cert-pop-completed.png)
 
+Note: If it is not possible for the admin to carry out this process himself for organizational reasons, the proof of possession for the corresponding certificate can be manually requested and the {{< product-c8y-iot >}} support team can complete the proof of possession through a backend API upon reasonable verification.
```
Comment (MWindel) [inline content/device-integration/mqtt-bundle/device-certificates.md:406]:
```suggestion
{{< c8y-admon-info >}}
If it is not possible for the administrator to carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.
{{< /c8y-admon-info >}}
```

**Comment by rssag** [comment]
which releases does this apply to? @vfo-softwareag

**Review by rssag — APPROVED** [review]

File: content/device-integration/mqtt-bundle/device-certificates.md [hunk 2]
```diff
@@ -403,6 +403,11 @@ The steps for the proof of possession are as follows:
 - The proof of possession is confirmed if the uploaded signed verification code matches the signed verification code expected by the platform:
 ![Proof of possession completed](/images/mqtt/devmgmt-mqtt-cert-pop-completed.png)
 
+{{< c8y-admon-info >}}
+If it is not possible for the administrator to carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.
```
Comment (BeateRixen) [inline content/device-integration/mqtt-bundle/device-certificates.md:407]:
```suggestion
If administrators cannot carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
> which releases does this apply to? @vfo-softwareag

As discussed with @vfo-softwareag in a chat, the target version is 10.16.

**Pull request merged**